docs/hazmat/primitives/asymmetric/ec.rst

```diff
@@ -122,6 +122,43 @@ Elliptic Curve Signature Algorithms
         :returns: A new instance of a :class:`EllipticCurvePublicKey`
             provider.
 
+Elliptic Curve Key Exchange algorithm
+-------------------------------------
+
+.. class:: ECDH(algorithm)
```

Comment:
Yes, I think that's a good summary.

Regarding the KDF, would it be acceptable to pass "None" explicitly as an argument (and not making it a default) instead of creating a NullKDF() function to pass in? I know I can do it by passing in "lambda x: x", but that's ugly.

Alternately, could NullKDF() be defined somewhere in "cryptography" (probably in hazmat.primitives.kdf, since it could be used in any place a KDF is expected)? That would be more readable than passing in None, and it would provide a standard name instead of each caller defining their own.